guides/keras_hub/getting_started.py

```diff
@@ -1,633 +1,788 @@
-    metrics=[keras.metrics.SparseCategoricalAccuracy()],
-    jit_compile=True,
+inputs = backbone.input
+x = backbone(inputs)
+x = LastTokenPooler(
+    name="pooler",
+)(x, inputs["padding_mask"])
+x = keras.layers.Dense(
+    2048,
+    activation="relu",
+    name="pooled_dense",
+)(x)
+x = keras.layers.Dropout(
+    0.1,
+    name="output_dropout",
+)(x)
+outputs = keras.layers.Dense(
+    2,
+    activation="softmax",
+    name="output_dense",
+)(x)
+text_classifier = keras.Model(inputs, outputs)
+text_classifier.summary()
+
+"""
+Before we train, there is one last trick we should employ to make this code run on free
+tier colab GPUs. We can see from our model summary our model takes up almost 10 gigabytes
+of space. An optimizer will need to make multiple copies of each parameter during
+training, taking the total space of our model during training close to 30 or 40
+gigabytes.
+
+This would OOM many GPUs. A useful trick we can employ is to enable LoRA on our
+backbone. LoRA is an approach which freezes the entire model, and only trains a low-parameter
+decomposition of large weight matrices. You can read more about LoRA in this [Keras
+example](https://keras.io/examples/nlp/parameter_efficient_finetuning_of_gpt2_with_lora/).
+Let's try enabling it and re-printing our summary.
+"""
+
+backbone.enable_lora(4)
+text_classifier.summary()
+
+"""
+After enabling LoRA, our model goes from 10GB of traininable parameters to just 20MB.
+That means the space used by optimizer variables will no longer be a concern.
+
+With all that set up, we can compile and train our model as normal.
+"""
+
+text_classifier.compile(
+    optimizer=keras.optimizers.Adam(5e-5),
+    loss="sparse_categorical_crossentropy",
+    metrics=["accuracy"],
 )
-model.fit(
-    imdb_preproc_train_ds,
-    validation_data=imdb_preproc_val_ds,
-    epochs=3,
+text_classifier.fit(
+    train_ds,
+    validation_data=val_ds,
 )
 
 """
-Excitingly, our custom classifier is similar to the performance of fine-tuning
-`"bert_tiny_en_uncased"`! To see the advantages of pretraining and exceed 90% accuracy we
-would need to use larger **presets** such as `"bert_base_en_uncased"`.
+We are able to achieve over ~93% accuracy on the movie review sentiment
+classification problem. This is not bad, given that we only used a 10th of our
+original dataset to train.
+
+Taken together, the `backbone` and `tokenizer` we created in this example
+allowed us access the full power of pretrained Gemma checkpoints, without
+restricting what we could do with them. This is a central aim of the KerasHub
+API. Simple workflows should be easy, and as you go deeper, you gain access to a
+deeply customizable set of building blocks.
+"""
+
+"""
+## Going further
+
+This is just scratching the surface of what you can do with the KerasHub.
+
+This guide shows a few of the high-level tasks that we ship with the KerasHub library,
+but there are many tasks we did not cover here. Try [generating images with Stable
```

Comment:
we can maybe link to this page too here to point to the full list of models to try out - https://keras.io/api/keras_hub/models/